src/main/java/com/google/jenkins/plugins/computeengine/AcceleratorConfiguration.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
I'm making a small tweak to these dates. I realized this while working on the google-storage-plugin PR, but this will be "2018-2019" instead of just "2018", to reflect both the original date and the date that it was updated.